UndertaleModTool/Scripts/Resource Unpackers/ExportSpecificSprites.csx

```diff
@@ -0,0 +1,67 @@
+//Basically a mashup of ExportAllSprites and DumpSpecificCode
+using System.Text;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+bool padded = (ScriptQuestion("Export all sprites WITH padding?"));
+
+string texFolder = PromptChooseDirectory();
+if (texFolder == null)
+    throw new ScriptException("The export folder was not set.");
+texFolder = texFolder + "Sprites" + Path.DirectorySeparatorChar;
+Directory.CreateDirectory(texFolder);
+
+TextureWorker worker = new TextureWorker();
+
+List<UndertaleSprite> spritesToDump = new List<UndertaleSprite>();
+List<String> splitStringsList = new List<String>();
+
+string InputtedText = "";
+InputtedText = SimpleTextInput("Menu", "Enter the name of the sprites", InputtedText, true);
+string[] IndividualLineArray = InputtedText.Split('\n', StringSplitOptions.RemoveEmptyEntries);
+foreach (var OneLine in IndividualLineArray)
+{
+    splitStringsList.Add(OneLine.Trim());
+}
+for (var j = 0; j < splitStringsList.Count; j++)

```

Comment:
why not foreach through the list?

Reply:
Honestly because I'm dumb and don't understand how to use foreach yet (any uses of foreach were already there when I adapted the code from ExportAllSprites or DumpSpecificCode

Reply:
basically, these two are basically equivalent:
for loop
```cs
for (int i = 0; i < myStringList.Count; i++)
{
  string listElement = myStringList[i];
  Console.WriteLine(listElement);
}
```
foreach loop
```cs
foreach (string listEelement in myStringList)
{
  Console.WriteLine(listElement);
}
```